style.css

```diff
@@ -0,0 +1,114 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  background: rgba(237, 255, 247, 1);
+  padding: 1rem;
+}
+
+.wrapper {
+  background-color: rgba(168, 192, 255, 0.17);
+  box-shadow: 0 0 1rem 0 rgba(0, 0, 0, 0.25);
+  margin: auto;
+  padding: 1rem;
+  width: 40vh;
+  height: auto;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.title {
+  margin-block-end: 1rem;
+  display: flex;
+  justify-content: center;
```

Comment:
Nu e total greșit să folosești flexul aici. 

Puteai să pui și clasicul `text-align: center`